### PR TITLE
feat(result-not-found): add result not found page

### PR DESCRIPTION
**Pull request by sriramanvellingiri**
#51 

gitscreenshot

### CODE REVIEW

File: react/src/components/blocks/ResultNotFound/ResultNotFound.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import ResultNotFoundImg from '../../../static/images/result_not_found.svg'
+
+function ResultNotFound(props) {
+
+  return (
+    <>
+      <p>Your search -{props.searchText} - did not match any documents.</p>
+      <p> Suggestions:</p>
+      <ul>
+            <li> Make sure that all words are spelled correctly. </li>
```
Comment (ramsunvtech) [inline react/src/components/blocks/ResultNotFound/ResultNotFound.js:13]: localize this

File: react/src/components/blocks/ResultNotFound/ResultNotFound.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import styles from './ResultNotFound.module.css'
+// image import
+import ResultNotFoundImg from '../../../static/images/result_not_found.svg'
+
+function ResultNotFound(props) {
+
+  return (
+    <>
+      <p>Your search -{props.searchText} - did not match any documents.</p>
```
Comment (ramsunvtech) [inline react/src/components/blocks/ResultNotFound/ResultNotFound.js:10]: destruct this like below

```
function ResultNotFound(props) {
const { searchText = '' } = props

  return (
    <>
      <p>Your search - {searchText} - did not match any documents.</p>
```

File: react/src/components/blocks/ResultNotFound/ResultNotFound.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      <p> Suggestions:</p>
+      <ul>
+            <li> Make sure that all words are spelled correctly. </li>
+            <li> Try different keywords. </li>
+            <li> Try more general keywords.</li>
+      </ul>
+      <img
+        src={ResultNotFoundImg}
+        alt='ResultNotFound'
```
Comment (ramsunvtech) [inline react/src/components/blocks/ResultNotFound/ResultNotFound.js:19]: localize

File: react/src/modules/SearchResults/SearchResults.js [hunk 4]
```diff
@@ -3,6 +3,7 @@ import React from 'react'
 // blocks
 import SearchForm from '@/blocks/SearchForm'
 import ResultList from '@/blocks/ResultList'
+import ResultNotFound from '@/blocks/ResultNotFound'
```
Comment (ramsunvtech) [inline react/src/modules/SearchResults/SearchResults.js:6]: rename it as `NoResulFound`

**Review by ramsunvtech — CHANGES_REQUESTED** [review]

File: react/src/modules/SearchResults/SearchResults.js (hunk 4 above)
Comment (sriramanvellingiri) [reply react/src/modules/SearchResults/SearchResults.js:6]: Done

File: react/src/components/blocks/ResultNotFound/ResultNotFound.js (hunk 1 above)
Comment (sriramanvellingiri) [reply react/src/components/blocks/ResultNotFound/ResultNotFound.js]: Done

**Review by ramsunvtech — APPROVED** [review]

**Pull request merged**